src/Forms/FormMessage.php

```diff
@@ -92,6 +91,57 @@ public function setMessage(
         return $this;
     }
 
+    /**
+     * Appends a message to the existing message if the types match, overrides if not
+     *
+     * @param string $message Message string
+     * @param string $messageType Message type
+     * @param string $messageCast
+     * @param bool $force Whether or not to override the existing message if appending is invalid
+     * @throws InvalidArgumentException
+     * @return $this
```

Comment:
```suggestion
     * @param bool $force Whether or not to override the existing message if appending is invalid
     * @throws InvalidArgumentException
```
We don't need to mention the return type or parameters in the PHPDoc unless they add additional context. Having them there increases the chance that they'll become desynced with the actual typehint.